streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -496,32 +530,20 @@ private void captureOutputRecords() {
             final String outputTopicName = record.topic();
             if (internalTopics.contains(outputTopicName) || processorTopology.sourceTopics().contains(outputTopicName)
                 || globalPartitionsByTopic.containsKey(outputTopicName)) {
-
-                final byte[] serializedKey = record.key();
-                final byte[] serializedValue = record.value();
-
-                pipeInput(new ConsumerRecord<>(
-                    outputTopicName,
-                    -1,
-                    -1L,
-                    record.timestamp(),
-                    TimestampType.CREATE_TIME,
-                    0L,
-                    serializedKey == null ? 0 : serializedKey.length,
-                    serializedValue == null ? 0 : serializedValue.length,
```

Comment:
@mjsax, What this means, is actions needed from my side

Reply:
No -- I just acknowledged that you basically move code that was already "wrong", so not your fault to use `0` instead of `Consumer.NULL_SIZE` :) -- no action needed.